Sync/metrics.yaml

```
@@ -20,10 +20,13 @@ sync:
       - temp-history-sync
       - temp-bookmarks-sync
       - temp-logins-sync
+      - temp-rust-tabs-sync
     bugs:
       - https://mozilla-hub.atlassian.net/browse/SYNC-3008
+      - https://mozilla-hub.atlassian.net/browse/SYNC-3237
     data_reviews:
       - https://github.com/mozilla-mobile/firefox-ios/pull/10353#issuecomment-1087532698
+      - https://github.com/mozilla-mobile/firefox-ios/pull/11285#issuecomment-1184886514
     data_sensitivity:
       - technical
```

To get this in line with policy, we should add 
```suggestion
      - technical
      - highly_sensitive
```